### PR TITLE
[KHRGA-59] Update "Events" from Runtime Classes

### PR DESCRIPTION
**Pull request by PetroBondar**
Issue: KHRGA-59

### CODE REVIEW

File: source/iface/event.rst [hunk 1]
```diff
@@ -33,39 +44,44 @@ kernel.
 ::
 
   event();
-  event(cl_event clEvent, const sycl::context& syclContext);
 
-Construct an event.
+Constructs an event that is immediately ready.
```
Comment (imvucic) [inline source/iface/event.rst]: should be ``event``...the same below

Comment (imvucic) [reply source/iface/event.rst]: I meant with quotes

File: source/iface/event.rst [hunk 2]
```diff
@@ -33,39 +44,44 @@ kernel.
...
-================
+The event has no dependencies and no associated commands.
 
-::
+Waiting on this event will return immediately and querying
+its status will return ``sycl::info::event_command_status::complete``.
 
-  cl_event get();
+The event is constructed as though it was created
+from a default-constructed queue.
```
Comment (imvucic) [inline source/iface/event.rst]: quotes on queue

File: source/iface/event.rst [hunk 3]
```diff
@@ -33,39 +44,44 @@ kernel.
...
+===============
 
 ::
 
-  bool is_host() const;
+  backend get_backend() const noexcept;
 
-
-Returns True if this a host event.
+Returns a backend identifying the SYCL backend associated with this event.
```
Comment (imvucic) [inline source/iface/event.rst]: quotes -> backend and event...I'm just following Description from rev8

**Comment by PetroBondar** [comment]
Merged into SYCL_Reference main.

**Pull request closed without merging**